source/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -331,33 +331,39 @@ specify(name, config, fn)
 
 {% partial allowed_test_config %}
 
-### Suite of test configuration
+### Suite configuration
 
-You can configure the size of the viewport height and width within a suite.
+If you want to target a suite of tests to run or be excluded when run in a specific browser, you can override the `browser` configuration within the suite configuration. The `browser` option accepts the same arguments as {% url "`Cypress.isBrowser()`" isbrowser %}.
 
 ```js
-describe('page display on medium size screen', {
-  viewportHeight: 1000,
-  viewportWidth: 400
-}, () => {
-  it('does not display sidebar', () => {
-    cy.get('#sidebar').should('not.be.visible')
+describe('When in Chrome', {  browser: '!chrome' } () => {
+  it('Shows warning', () => {
+    cy.get('.browser-warning')
+      .should('contain', 'For optimal viewing, use Chrome browser')
   })
 
-  it('shows hamburger menu', () => {
-    cy.get('#header').find('i.menu').should('be.visible')
+  it('Links to browser compatibility doc', () => {
+    cy.get('a.browser-compat')
+      .should('have.attr', 'href')
+      .and('include', 'browser-compatibility)
   })
 })
 ```
 
 ### Single test configuration
 
-If you want to target a test to run or be excluded when run in a specific browser, you can override the `browser` configuration within the test configuration. The `browser` option accepts the same arguments as {% url "`Cypress.isBrowser()`" isbrowser %}.
+You can configure the number of times to retry a test if they fail during `cypress run` or `cypress open` separately. See {% url "Test Retries" test-retries %} for more information.
```

Comment:
Pluralization mismatch. "separately" is unneeded. It was referencing the tests failing separately instead of the configuration.

```suggestion
You can configure the number of times to retry tests if they fail during `cypress run` or `cypress open`. See {% url "Test Retries" test-retries %} for more information.
```